### ISSUE TITLE
Ability to turn off bot's camera after sending content to the meeting

### ISSUE THREAD

**Issue by maimar-sw** [microsoftgraph/microsoft-graph-comms-samples#391]
I would like the bot to display its avatar after using the audio and video send feature. Is there any way to accomplish this? So far, the camera displays the last sent frame. Thanks!

**Comment by philomathikus** [comment]
@maimar-sw Btw, were you able to send H264 via the bot? So far, we've only been able to send NV12 video.

**Comment by maimar-sw** [comment]
@philomathikus Not really, I stopped trying to send H264 as sending NV12 is not blocking me, are you really needing it for your use case?

**Comment by philomathikus** [comment]
@maimar-sw The goal is to forward a real-time H264 feed generated from a different device to Teams via the Bot. We have a decoding engine that we could implement in the bot to feed it NV12 frames. However, it would obviously decrease performance... Further, since the receiving and sending events are blocking operations, it might lead to other issues (should we be forced to decode and then have the bot re-encode it).

**Comment by maimar-sw** [comment]
I understand, did you check [this](https://github.com/microsoftgraph/microsoft-graph-comms-samples/issues/24#issuecomment-437185877) answer?

**Comment by philomathikus** [comment]
Yes, we are aware of that thread but it's not really pointing us to a solution. We always get some error from the Microsoft.Skype.Internal.Media.H264.dll which we can't obviously debug in more detail...

**Comment by ssulzer** [comment]
@maimar-sw 
Right now there's no way I'm aware of for a bot to indicate it has stopped sending video. This would require new Graph APIs.
